lib/Service/ObjectService.php

```diff
@@ -91,11 +191,11 @@ public function saveObject($register, $schema, array $object): ObjectEntity
 	 *
 	 * @return ObjectEntity The resulting object.
 	 */
-	public function getObject(Register $register, string $uuid): ObjectEntity
+	public function getObject(Register $register, Schema $schema, string $uuid): ObjectEntity
 	{
 		// Lets see if we need to save to an internal source
-		if ($register->getSource() === 'internal') {
-			return $this->objectEntityMapper->findByUuid($register,$uuid);
+		if ($register->getSource() === 'internal' || $register->getSource() === '') {
```

Comment:
in_array would be shorter
in_array($register->getSource(), ['internal', ''])

Or maybe use empty($register->getSource()) instead of $register->getSource() === ''

Reply:
But not necessarily more readable. This also is temporary until we can choose sources correctly